src/components/FullPlayground.tsx

```diff
@@ -922,115 +890,110 @@ export function ThemedAppView(props: { datastore: DataStore }) {
               </div>
             )}
 
-            {currentItem?.kind === DataStoreItemKind.ASSERTIONS && (
-              <ValidateButton
-                datastore={datastore}
-                validationState={validationState}
-                conductValidation={conductValidation}
-                developerService={developerService}
-              />
-            )}
-
-            {currentItem?.kind === DataStoreItemKind.EXPECTED_RELATIONS && (
-              <ValidateButton
-                datastore={datastore}
-                validationState={validationState}
-                conductValidation={conductValidation}
-                developerService={developerService}
-              />
-            )}
+            {currentItem?.kind &&
+              [
+                DataStoreItemKind.ASSERTIONS,
+                DataStoreItemKind.EXPECTED_RELATIONS,
+              ].includes(currentItem.kind) && (
```

Comment:
Modified the conditional to be this rather than separate conditionals that render the same thing